emulators/noah/Portfile

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               GitHub 1.0
+
+name    noah
+subport noahstrap {}
+
+categories              emulators
+maintainers             nomaintainer
+platforms               darwin
+
+license                 Dual MIT/GPL
```

Comment:
We need to clean up the exact license here. I think it's GPL2+ and I'm looking for the MIT part. We need to use exact MacPorts license words, which I can never find when I'm looking for them, but there is a wiki where they are all listed.

Reply:
That would probably be `license {MIT GPL-2+}` then. However I did not see whether the noah project uses "GPLv2 only" or "GPLv2 or later". That is a detail the project authors might need to consider/specify more carefully.